hydra-redis/collections_endpoint.py

```diff
@@ -0,0 +1,161 @@
+#                self.redis_graph.commit()
+                # put/create the graph in redis
+                print(
+                    "property of obj which can be class",
+                    obj_properties["class_property"],
+                    "collection"
+                )
+
+                if obj_properties_classlist:
+                    c = Classes(self.redis_graph)
+                    c.objects_property(
+                        objects_node,
+                        obj_properties_classlist,
+                        url_node,
+                        api_doc)
+        else:
+            print("NO MEMBERS")
+
+    def endpointCollection(
+            self,
+            collection_endpoint,
+            url_node,
+            api_doc,
+            base_url):
+        """It makes a node for every collection endpoint."""
+        print("accessing every collection in entrypoint")
+        for obj in collection_endpoint:
+            obj_method = []
+            properties = []
+            obj_properties = {}
+            print(
+                "check url for endpoint",
+                base_url +
+                collection_endpoint[obj])
+            new_url = base_url + collection_endpoint[obj]
+            # url for every collection endpoint
+            for obj2 in api_doc.collections[
+                                            obj][
+                                            "collection"].supportedOperation:
+                obj_method.append(obj2.method)
+            obj_properties["methods"] = str(obj_method)
+            # all the operations for the collection endpoint is stored in
+            # method
+            print(obj_properties["methods"])
+            for obj2 in api_doc.collections[
+                                            obj][
+                                            "collection"].supportedProperty:
+                properties.append(str(obj2.desc[4:]))
+            obj_properties["title"] = str(properties)
+            response = ur.urlopen(base_url + collection_endpoint[obj])
+            new_file = json.loads(response.read().decode('utf-8'))
+            # retrieving the objects from the collection endpoint
+            obj_properties["members"] = str(new_file["members"])
+            obj_collection_node = Node(
+                label="collection", alias=obj, properties=obj_properties)
+            self.redis_graph.add_node(obj_collection_node)
+            edge = Edge(url_node, "has_collection", obj_collection_node)
+            self.redis_graph.add_edge(edge)
+            # set an edge between the entrypoint and collection endpoint
+#            print("commit endpoint collection")
+#            self.redis_graph.commit()
+            # create the graph in redis
+            self.collectionobjects(
+                obj_collection_node,
+                new_file["members"],
+                new_url,
+                url_node,
+                api_doc,
+                base_url
```

Comment:
**I'm not reviewing this file right now but most of the pointers for classes_endpoint.py go for this as well.**